modules/sdp/LocalSdpMunger.js

```diff
@@ -207,6 +211,28 @@ export default class LocalSdpMunger {
                     const streamAndTrackIDs = ssrcLine.value.split(' ');
 
                     if (streamAndTrackIDs.length === 2) {
+                        // FIXME
+                        // The problem here is that Chrome in unified gives `-` as the stream ID.
```

Comment:
Had to workaround a problem here for the POC. Another thing to address as part of the project.